ai_edge_torch/convert/test/test_convert.py

```diff
@@ -306,6 +310,99 @@ def forward(self, x, y):
         model_coverage.compare_tflite_torch(edge_model, model, args_gen, kwargs_gen)
     )
 
+  def test_convert_model_with_args_nested_kwargs(self):
+    """
+    Test converting a simple model with both sample_args and nested sample_kwargs.
+    """
+
+    @dataclass
+    class TestContainer:
+      data_1: torch.Tensor
+      data_2: Tuple[torch.Tensor, torch.Tensor]
+
+    torch.export.register_dataclass(TestContainer, serialized_type_name="TestContainer")
+
+    #### case 1
+    class SampleModel1(torch.nn.Module):
+
+      def forward(self, x: torch.Tensor, y: torch.Tensor, z: TestContainer):
+        return x + y + z.data_1 + z.data_2[0] + z.data_2[1]
+
+    args = (torch.randn(10, 10),)
+    kwargs = dict(
+        y=torch.randn(10, 10),
+        z=TestContainer(
+            data_1=torch.randn(10, 10),
+            data_2=(torch.randn(10, 10), torch.randn(10, 10)),
+        ),
+    )
+    flat_inputs = {
+        "args_0": args[0].numpy(),
+        "y": kwargs["y"].numpy(),
+        "z_data_1": kwargs["z"].data_1.numpy(),
+        "z_data_2_0": kwargs["z"].data_2[0].numpy(),
+        "z_data_2_1": kwargs["z"].data_2[1].numpy(),
+    }
+    self._args_kwargs_test_helper(SampleModel1(), args, kwargs, flat_inputs)
+
+    #### case 2
```

Comment:
pls separate this into multiple tests. Name them `test_convert_model_with_args_nested_kwargs_{n}` should be good enough.

Reply:
Done.